rust/src/rdp/log.rs

```diff
@@ -25,7 +25,7 @@ use std;
 use x509_parser::parse_x509_der;
 
 #[no_mangle]
-pub extern "C" fn rs_rdp_to_json(tx: *mut std::os::raw::c_void) -> *mut JsonT {
+pub unsafe extern "C" fn rs_rdp_to_json(tx: *mut std::os::raw::c_void) -> *mut JsonT {
```

Comment:
The linter is probably not complaining about these, since they're exposing an interface for C to call.

Reply:
The linter was complaining about this function as it dereferences a raw pointer, and suggest the function should be unsafe, not just block.